src/lobsterpy/cli.py

```diff
@@ -472,6 +487,128 @@ def get_parser() -> argparse.ArgumentParser:
     )
 
+    # Args specific to icohp distance plotter args
+    icohp_distance_plotter_args = argparse.ArgumentParser(add_help=False)
+    icohp_distance_plotter_group = icohp_distance_plotter_args.add_argument_group(
+        "Options specific to ICOHP distance plotter"
+    )
+    icohp_distance_plotter_group.add_argument(
+        "-alpha",
+        type=float,
+        default=0.4,
+        dest="alpha",
+        help="Transparency of the markers in the plot.",
+    )
+    icohp_distance_plotter_group.add_argument(
+        "-cbonds",
+        "--colorbonds",
+        "--color-bonds",
+        default=False,
+        dest="colorbonds",
+        action="store_true",
+        help="If set, will color ICOHPs based on atom types",
+    )
+    icohp_distance_plotter_group.add_argument(
+        "-c",
+        "--colors",
+        dest="colors",
+        nargs="+",
+        type=str,
+        default=None,
+        help="Custom list of colors for the plot. "
+        "Should be equal to the number of unique atom pairs in the structure.",
+    )
+    icohp_distance_plotter_group.add_argument(
+        "-lprefix",
+        "--legendprefix",
+        "--legend-prefix",
+        type=str,
+        dest="legendprefix",
+        default="",
+        help="Prefix for the legend in the plot.",
+    )
+    icohp_distance_plotter_group.add_argument(
+        "-msize",
+        "--markersize",
+        "--marker-size",
+        type=float,
+        default=50,
+        dest="markersize",
+        help="Size of the markers in the plot.",
+    )
+    icohp_distance_plotter_group.add_argument(
+        "-mstyle",
+        "--markerstyle",
+        "--marker-style",
+        type=str,
+        default="o",
+        dest="markerstyle",
+        help="Marker style for the plot.",
+    )
+
+    # Args specific to plotting BWDFs
+    bwdf_plotting_args = argparse.ArgumentParser(add_help=False)
+    bwdf_plotting_group = bwdf_plotting_args.add_argument_group("Options specific to plotting BWDFs")
+    bwdf_plotting_group.add_argument(
+        "-atompairs",
+        "--atom-pairs",
+        dest="atompairs",
+        action="store_true",
+        default=False,
+        help="If True, will plot the BWDFs for all unique atom pairs. Default: False.",
+    )
+    bwdf_plotting_group.add_argument(
+        "-binwidth",
+        "--bin-width",
+        type=float,
+        dest="binwidth",
+        default=0.02,
+        help="Bin width used for computing the BWDFs. Default: 0.02.",
+    )
+    bwdf_plotting_group.add_argument(
+        "-interacttol",
+        "--interactions-tolerance",
+        type=float,
+        dest="interacttol",
+        default=1e-3,
+        help="Numerical tolerance considered for interactions to be insignificant. Default: 1e-3.",
+    )
+    bwdf_plotting_group.add_argument(
+        "-maxlen",
+        "--maxlength",
+        "--max-length",
+        type=float,
+        dest="maxlen",
+        default=5,
```

Comment:
if 6.0 Angstrom is the default cutoff for creating lobsterin (?), maybe let's use 6.0 as default here as well?

Reply:
Done